src/site/_filters/find-tags.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const blogTags = require("../_data/blogTags");
+
+/**
+ * @param {Array<string>} tags Array of tags.
+ * @return {Array<object>} An array of tags urls, titles, and any other information.
+ */
+module.exports = (tags) => {
+  const validTags = [];
+
+  for (const tag of tags) {
+    const lowerCaseTag = tag.toLowerCase();
+    const tagData = blogTags[lowerCaseTag];
+    if (tagData) {
+      tagData.url = `/tags/${lowerCaseTag}/`;
+      validTags.push(tagData);
+    }
```

Comment:
Should it be an error to specify an invalid tag?
(It could be an error in a bunch of places, but this seems like as good as any)

Reply:
Will remove `toLowerCase()`. However I don't think we should consider it an error if the tag is invalid, as there are a lot of tags that old posts have used tags that aren't in `blogData`

Reply:
I think this may be worth discussing more.

I'm in favor of having the site error if someone uses an unsupported tag for two reasons:
1. If they add a tag and the site silently fails to display it, they might be confused.
2. We don't want folks arbitrary expanding the tags grammar.

If we need to go back and update old posts then we can do that with the help of the writers. They may want us to pursue the tags synonyms thing mentioned in https://github.com/GoogleChrome/web.dev/issues/1808#issuecomment-548617577 but I think as a v1 of the feature we should just focus on consolidating the existing list of tags. Then we can pursue the synonym thing as a future update once we've all had a chance to debate it more.

Reply:
I believe @robdodson and I agreed to allow it to silently fail for now, and at a later point we will address this via linting.